### PR TITLE
Updates and some small improvements

### PR DESCRIPTION
**Pull request by invidian**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
 
       - name: Run lint
-        run: make lint
+        run: make lint check-update-linters
```
Comment (knrt10) [inline .github/workflows/ci.yaml:40]: Shouldn't this be part of previous commit as the target was introduced then?

**Review by knrt10 — COMMENTED** [review]
Just a question, rest LGTM

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (invidian) [reply .github/workflows/ci.yaml:40]: I could probably merge it with previous commit or split out changes to `.golangci.yml` file into yet separate commit to keep the line dependencies chain (next commit depending on the previous one, which happens all the time anyway), but given that this most likely won't be bisected or reverted (changes are rather trivial) and commits are very small anyway, I think any combination is probably acceptable.

**Review by knrt10 — APPROVED** [review]
+1

**Pull request merged**